### ISSUE TITLE
Not compiling on Android 12 and +

### ISSUE THREAD

**Issue by salim-lachdhaf** [SocketMobile/samples-android#1]
android:exported needs to be explicitly specified for element <receiver#com.socketmobile.capture.socketcam.client.CaptureExtension$ConnectionStateReceiver>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.

**Comment by doubleo2** [comment]
@salim-lachdhaf Until a fix is released, you can work around the issue by adding the following to your manifest

```xml
<manifest ...>
    <application ...>
        <receiver android:name="com.socketmobile.capture.socketcam.client.CaptureExtension$ConnectionStateReceiver"
            android:exported="true"
            tools:replace="android:exported" />
    </application>
</manifest>
```

**Comment by chitra-socket** [comment]
Issue is fixed with version 1.6.5. Thanks!
